src/main/java/org/openrewrite/staticanalysis/EqualsAvoidsNullVisitor.java

```diff
@@ -17,7 +17,7 @@
 
 import lombok.EqualsAndHashCode;
 import lombok.Value;
-import org.jspecify.annotations.Nullable;
+import lombok.val;
```

Comment:
Lets not use `lombok.val`; we even have a recipe to phase out it's use. :)
https://docs.openrewrite.org/recipes/java/migrate/lombok/lombokvaltofinalvar

Reply:
Yes, I should have thought that the opposite approach is actually better, as it avoids unnecessary clutter by adopting the Kotlin approach. However, I will, of course, obey the common style of writing imperative and boilerplate code instead of small functions.